library/src/main/java/hotchemi/android/rate/DialogManager.java

```diff
@@ -18,9 +18,20 @@ private DialogManager() {
 
     static Dialog create(final Context context, DialogOptions options) {
         AlertDialog.Builder builder = getDialogBuilder(context);
-        builder.setMessage(options.getMessageResId());
+        if(options.getMessageText() == null) {
```

Comment:
What do you mean by that exactly? Less if statements? I cannot do that without changing DialogOptions class methods and use those if statements there...